.travis.yml

```diff
@@ -10,91 +10,119 @@ before_install:
   - sleep 15
   - sudo apt-get install -y
   - sudo apt-get install -y curl
+  - sudo apt-get install -y shellcheck
+
+language: shell
+  script:
+    - bash -c 'shopt -s globstar nullglob; shellcheck **/*.{sh,ksh,bash}'
```

Comment:
```shellcheck **/*.{sh,ksh,bash}``` will lookup for scripts  in all directories and check their syntax . Why did you add an extra line (line 18) to list the scripts..?

Reply:
As when I was looking about shellcheck with Travis , found this https://github.com/koalaman/shellcheck/wiki/TravisCI and I was unsure about globstar and nullglob so for confirmation I added the extra line for all the scripts , think I should remove line 18 , as line 17 is already doing shellcheck on all the scripts.

Reply:
Can you please check the indentation , I think script should be sibling of language not its child?

Reply:
In scripts, the standard way is to provide only 2 spaces in the beginning of the child block. Can you please provide only 2 spaces in beginning of line 17.